llvm/lib/Support/TimeProfiler.cpp

```diff
@@ -79,15 +79,15 @@ struct TimeTraceProfilerEntry {
   // Calculate timings for FlameGraph. Cast time points to microsecond precision
```

Comment:
Can you split the changes to this file into a separate PR? Also check the file, the comment still states microseconds.

Reply:
Removed this file from this PR